collections.json

```diff
@@ -82,5 +82,45 @@
                 ]
             }
         ]
-    }
+    },
+    {
+    "project": "Professor Cardano",
+    "collections": [
+        {
+            "logoUrl": "https://ipfs.io/ipfs/QmPriKLfgJA7HeR9uj38ij7YirmNyYYrqFNVCGZ6GCyqmQ?filename=Professor_Cardano_Lab.jpg",
+            "collectionName": "Professor Cardano & Professor Cardano BANG",
+            "policies": [
+              "e85a0209a5d4dd2f4bc12783f65742f1994b16e506eef8b7296fc778"
+            ]
+        },
+        {
+            "logoUrl": "https://ipfs.io/ipfs/Qmd9eQMsbLnjBc1qcZ9Td5qzzSW4Hz16naTi4gEUimETSg?filename=118%20-%20Oganesson%20%5BOg%5D.jpg",
+            "collectionName": "118 - Oganesson [Og]",
+            "policies": [
+              "c2fe5590c582d1c53dbd7dd72b702b78407a67dae9a2046d577fa4c3"
+            ]
+        }
+         {
```

Comment:
I just noticed now that you're also missing a `,` here and in some other places below